Makefile

```
@@ -4,6 +4,8 @@ OUTPUT_DIRNAME		?= output
 DOC_FILENAME		?= oci-runtime-spec
 DOCKER			?= $(shell command -v docker 2>/dev/null)
 PANDOC			?= $(shell command -v pandoc 2>/dev/null)
+
+PANDOC_CONTAINER ?= quay.io/oci/pandoc:1.17.0.3-2.fc25.x86_64
```

```suggestion
PANDOC_IMAGE ?= quay.io/oci/pandoc:1.17.0.3-2.fc25.x86_64
```

:innocent: 

Given this version string is now much more complex than trusting `vbatts/pandoc:latest`, would it also make sense to either have a `docker pull $(PANDOC_IMAGE)` target or perhaps have `install.tools` do that?

---

(To avoid the `.travis.yml` duplication, which will inevitably get out of sync or trip folks up if they need to bump this for whatever reason)

---

`quay.io/oci/pandoc:latest` seems to work fine.  Is there any reason for using an older version?